plugins/catalog-backend-module-incremental-ingestion/README.md

```diff
@@ -121,7 +121,7 @@ To create an Incremental Entity Provider, you need to know how to retrieve a sin
 Here is the type definition for an Incremental Entity Provider.
 
 ```ts
-interface IncrementalEntityProvider<TCursor, TContext> {
+interface IncrementalEntityProvider<Cursor, Context> {
```

Comment:
Convention is to use the `T` prefix for type parameters when defining a generic:
```suggestion
interface IncrementalEntityProvider<TCursor, TContext> {
```